src/pgcopy/src/copy.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+syntax = "proto3";
+
+package mz_pgcopy.copy;
+
+message ProtoCopyFormatParams {
+    oneof kind {
+        ProtoCopyTextFormatParams text = 1;
+        ProtoCopyCsvFormatParams csv = 2;
+    }
+}
+
+message ProtoCopyTextFormatParams {
+    string null = 1;
+    string delimiter = 2;
+}
+
+message ProtoCopyCsvFormatParams {
+    uint32 delimiter = 1;
+    uint32 quote = 2;
+    uint32 escape = 3;
```

Comment:
The docs say these are all one-byte chars. Can we refactor things to make them all u8s?

Reply:
The refactor can be temporary and just happen to and from proto, then we fix it more fully later on. Well, maybe this doesn't matter because we can change the protos and don't need to worry about migration because both marshal and unmarshal sides will be running the same version.

Reply:
Sounds good 👍

Reply:
Looks like there's no uint8 for protobuf though https://protobuf.dev/programming-guides/proto3/#scalar

Reply:
whoa til!

Reply:
> whoa til!

same!